leapp/messaging/__init__.py

```diff
@@ -58,6 +59,13 @@ def stored(self):
         """
         return self._stored
 
+    def dialogs(self):
+        """
+        Gets all dialogs actually encountered during workflow run
```

Comment:
```suggestion
        Gets all dialogs actually encountered during workflow run
        
```

Please keep the required format of docstring:
```
One line title

N lines of whatever after
the empty line.
```

Reply:
You mean the newline missing between description\return? Ok, will fix that everywhere in the file.

Reply:
Yes, thanks.